### PR TITLE
DynamoDB: Add table loader for full-load operations

### PR DESCRIPTION
**Pull request by amotl**
## About
Bring DynamoDB full-load to Toolkit's `ctk load table` interface. 

## Documentation
https://cratedb-toolkit--226.org.readthedocs.build/io/dynamodb/loader.html

## Status
Alpha. For now, the implementation uses the same easy strategy to converge the source record into a single `data (OBJECT)` column in CrateDB. The 1:1 strategy may follow.

## Backlog
- GH-231

/cc @hammerhead, @zolbatar

### CODE REVIEW

File: cratedb_toolkit/io/dynamodb/copy.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            logger.info(f"Number of records written: {records_out}")
+            if records_out < records_in:
+                logger.warning("No data has been copied")
+
+    def items_to_sql(self, items):
+        """
+        Convert data for record items to INSERT statements.
+        """
+        for item in items:
+            yield self.translator.to_sql(item)
```
Comment (amotl) [inline cratedb_toolkit/io/dynamodb/copy.py:75]: That's another item transformation idea picked up from an example program. Please advise if this is sensible in all situations, or if it's just a special case.

```python
if 'id' in item and not isinstance(item['id'], str):
    item['id'] = str(item['id'])
```

/cc @wierdvanderhaar

File: cratedb_toolkit/io/dynamodb/backlog.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# DynamoDB Backlog
+
+- Pagination / Batch Getting.
+  https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/programming-with-python.html#programming-with-python-pagination
+
+- Use `batch_get_item`.
+  https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb/client/batch_get_item.html
+
+- Scan by query instead of full.
```
Comment (amotl) [inline cratedb_toolkit/io/dynamodb/backlog.md:9]: @wierdvanderhaar: With respect to alternative implementations, using batched reading from DynamoDB is probably way to go when processing large amounts of data?

Comment (wierdvanderhaar) [reply cratedb_toolkit/io/dynamodb/backlog.md:9]: Yes, we should use the batched method. Let's use a default batch size of 100 but give the option to use different batch sizes.

Comment (amotl) [reply cratedb_toolkit/io/dynamodb/backlog.md]: Hi. I added the remnant items to the backlog.

- GH-231

Do you agree to merge and release it first, following the "first make it work, then make it fast|beautiful|robust" paradigm, in order to get it out as quickly as possible?

**Review by wierdvanderhaar — APPROVED** [review]

**Pull request merged**